pkg/controller/controlplanecerts/controlplanecerts_controller.go

```diff
@@ -357,6 +330,41 @@ func (r *ReconcileControlPlaneCerts) generateControlPlaneCertsSyncSet(cd *hivev1
 	return syncSet, nil
 }
 
+func (r *ReconcileControlPlaneCerts) getServingCertificatesJSONPatch(cd *hivev1.ClusterDeployment, cdLog log.FieldLogger) (string, error) {
+	var buf strings.Builder
+
+	additionalCerts := cd.Spec.ControlPlaneConfig.ServingCertificates.Additional
+	if cd.Spec.ControlPlaneConfig.ServingCertificates.Default != "" {
+		cdLog.Debug("setting default serving certificate for control plane")
+
+		apidomain, err := r.defaultControlPlaneDomain(cd)
+		if err != nil {
+			cdLog.WithError(err).Error("failed to get control plane domain")
+			return "", err
+		}
+
+		cpCert := hivev1.ControlPlaneAdditionalCertificate{
+			Name:   cd.Spec.ControlPlaneConfig.ServingCertificates.Default,
+			Domain: apidomain,
+		}
+		additionalCerts = append([]hivev1.ControlPlaneAdditionalCertificate{cpCert}, additionalCerts...)
+	}
+	for i, additional := range additionalCerts {
+		if i > 0 {
+			buf.WriteString(",")
+		}
+		bundle := certificateBundle(cd, additional.Name)
+		cdLog.WithField("name", additional.Name).Debug("adding named certificate to control plane config")
+		buf.WriteString(fmt.Sprintf(` { "names": [ "%s" ], "servingCertificate": { "name": "%s" } }`,
+			additional.Domain, remoteSecretName(bundle.CertificateSecretRef.Name, cd)))
+	}
+
+	var kubeAPIServerNamedCertsTemplate = `[ {"op": "replace", "path": "/spec/servingCerts/namedCertificates", "value": [ %s ] } ]`
```

Comment:
```suggestion
	var kubeAPIServerNamedCertsTemplate = `[ { "op": "replace", "path": "/spec/servingCerts/namedCertificates", "value": [ %s ] } ]`
```
this was in consistent with other json patch template being added